### DISCUSSION TITLE
im a noob and cant get the docker compose command to work

### DISCUSSION THREAD

**Discussion by madmike970** [open-webui/open-webui#673] [General]
so i had ollama set up as a vm in proxmox and couldnt get the webui installed on that system.. just started a bare ubuntu 23.1 vm and installed docker with
apt-get docker.io
tried to run 
docker compose up -d --build
and got back
unknown shorthand flag: 'd' in -d
See 'docker --help'
i have no clue what im doin wrong any help is appreciated.. im just a construction worker trying to figure this stuff out lol

**Comment by justinh-rahb** [comment]
Did you install Docker and Compose according to the official docs? They start off with `apt remove docker.io docker-doc docker-compose docker-compose-v2 podman-docker containerd runc` to remove the distro-packaged versions of Docker and its dependencies or alternative drop-in compatibles.

**Comment by mhussaincov94** [comment]
hi! sir newb sir!
try this for size.
sudo docker-compose up -d --build
there is a - between docker and compose
don't go removeing packages yet.
tell me if that works.

**Comment by justinh-rahb** [comment]
> hi! sir newb sir! try this for size. sudo docker-compose up -d --build there is a - between docker and compose don't go removeing packages yet. tell me if that works.

In older versions of Compose, yes you'd invoke it with `docker-compose`. Not so with newer versions since v2, ollama-webui expects version `>=3.8` btw.
